### ISSUE TITLE
Problem with update of Munin graphics 2.0.25-1

### ISSUE THREAD

**Issue by BadBlockDev** [munin-monitoring/munin#559]
Hello,

On my machine with 13 infrastructure we migrated to Debian 8 and we have chosi of Munin used for monitoring our infrastructure but the problem is that graphics will update only once in 4-5 hours which is not acceptable. Even watching tasks cron executes the tasks in 5 minutes except that the graphics on the web interface to update 4h later.

In the old version no problem and now after a full update problem. Same problem on multiple machines.

We also get errors like this:

2015/09/06 4:55:08 [ERROR] In RRD: Error updating /var/lib/munin/Dev/Dev-http_loadtime-loadtime-g.rrd: / var / lib / munin / Dev / Dev-http_loadtime- loadtime-g.rrd: we found extra data update argument:

I reinstalled Munin almost 20 times but still the problem.

```
# Sendmail_mailqueue:
# Non-zero exit During autoconf (255)
# Sendmail_mailstats:
# Non-zero exit During autoconf (255)
# Sendmail_mailtraffic:
# Non-zero exit During autoconf (255)
# Tomcat_:
# Non-zero exit During autoconf (127)
# Varnish_:
# Non-zero exit During autoconf (2)
ns331359 root @: / var / log / munin munin-node # nano-configure.log
  GNU nano 2.2.6 File: munin-node-configure.log

# Non-zero exit During autoconf (255)
# Postgres_size_:
# Non-zero exit During autoconf (255)
# Postgres_transactions_:
# Non-zero exit During autoconf (255)
# Postgres_tuples_:
# Non-zero exit During autoconf (255)
# Postgres_users:
# Non-zero exit During autoconf (255)
# Postgres_xlog:
# Non-zero exit During autoconf (255)
# Proc:
# In family 'auto' goal Does not have 'autoconf' capability
# Sendmail_mailqueue:
# Non-zero exit During autoconf (255)
# Sendmail_mailstats:
# Non-zero exit During autoconf (255)
# Sendmail_mailtraffic:
# Non-zero exit During autoconf (255)
# Tomcat_:
# Non-zero exit During autoconf (127)
# Varnish_:
# Non-zero exit During autoconf (2)
```

I is not what you have done with updating but Munin is rendered unusable.

Translated by Google Translate

**Comment by ssm** [comment]
Please check that you do not have a parallel installation of munin in /usr/local/ or somewhere else on the perl module path.

**Comment by 0dataloss** [comment]
Hi, I have found the same problem myself with a new installation of Debian8
If you run the command

`munin-run http_loadtime`

You will see the following output
loadtime.value /etc/munin/plugins/http_loadtime: line 74: --quiet: command not found

This is because the script is trying to execute the command
loadtime=$(cd $TEMPO_DIR && $time_bin --quiet -f "%e" wget $wget_opt $target 2>&1)

With
time_bin=`which time`
Not sure why but the command 

`which time`

Output an empty string (even if the binary is installed and working correctly); only after "installing" the package with `apt-get install time` the system start working fine

**Comment by indywidualny** [comment]
So solution is actually simple. Dependency of munin-node is `time`. It should be added to dependencies while installing and all fine :)

**Comment by sumpfralle** [comment]
The "time" dependency issue is fixed in #886.

The original issue may be different, but lacks the details necessary for debugging. Please open another issue or ask via IRC or on the mailinglist in case of doubt.
